config/mycnf/test-suite.cnf

```diff
@@ -23,3 +23,4 @@ sql_mode = STRICT_TRANS_TABLES
 
 # set a short heartbeat interval in order to detect failures quickly
 slave_net_timeout = 4
+super-read-only = false
```

Comment:
test-suite is used only in vttestserver. VtTestserver uses vtcombo which has one sql instance for all vttablet. We cannot turn 'on/'off' super-read-only because it will make vtcombo confuse due to one instance of mysql for all vttablets.

Reply:
Are we using vttestserver in our e2e , i don't see instances where we are...